### ISSUE TITLE
Astro CLI --> When executing deploy command with invalid env file path not working as expected

### ISSUE THREAD

**Issue by sanchit135** [astronomer/astro-cli#874]
**Describe the bug**

```
Building image...
[+] Building 1.3s (11/11) FINISHED                                              
 => [internal] load build definition from Dockerfile                       0.0s
 => => transferring dockerfile: 85B                                        0.0s
 => [internal] load .dockerignore                                          0.0s
 => => transferring context: 83B                                           0.0s
 => [internal] load metadata for quay.io/astronomer/astro-runtime:6.0.4    1.1s
 => [1/1] FROM quay.io/astronomer/astro-runtime:6.0.4@sha256:06e2598d0fca  0.0s
 => [internal] load build context                                          0.0s
 => => transferring context: 21.79kB                                       0.0s
 => CACHED [2/1] COPY packages.txt .                                       0.0s
 => CACHED [3/1] RUN if [[ -s packages.txt ]]; then     apt-get update &&  0.0s
 => CACHED [4/1] COPY requirements.txt .                                   0.0s
 => CACHED [5/1] RUN if grep -Eqx 'apache-airflow\s*[=~>]{1,2}.*' require  0.0s
 => CACHED [6/1] COPY --chown=astro:astro . .                              0.0s
 => exporting to image                                                     0.0s
 => => exporting layers                                                    0.0s
 => => writing image sha256:12bada03db7516fc85ef825add9aa91492dc24c03ba2d  0.0s
 => => naming to docker.io/astral-terminator-8164/airflow:latest           0.0s

Use 'docker scan' to run Snyk tests against images to find vulnerabilities and learn how to fix them
Runtime Version: 6.0.4
testing astral-terminator-8164/airflow:latest
Pushing image to Astronomer registry
The push refers to repository [images.astronomer-stage.cloud/ckon5obsu03322822nvm6yumk/clai5znlk253991bd26e0ezfdwo]
190af4146669: Preparing 
622616ed72c7: Preparing 
364155b823e3: Preparing 
04e8d3bf88ac: Preparing 
14cf2d384316: Preparing 
5f70bf18a086: Preparing 
f9255478dbd6: Preparing 
cba3e953422e: Preparing 
abbe316e80c7: Preparing 
0d2d2efe9623: Preparing 
c3274ae0d2d1: Preparing 
eb1f55d33d3d: Preparing 
d4cd3f2e4977: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
b4db5f271750: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
6c1efa5db68c: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
07cf2b5e7e97: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
7d0fd9e54092: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
4b939f5c835f: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
48eb6a220e75: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
905bb40254e1: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
403990d16e20: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
bec8247b4a2e: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
840fead04799: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
89fbe0f67e30: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
f3a4a2b3bca3: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
c86f1430c8b4: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
b9e85b58b83f: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
466ee1641be0: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
150de0b737bd: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
00c1be188a46: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
5772b8372e59: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
a12586ed027f: Mounted from ckon5obsu03322822nvm6yumk/clai5qjb4219242bd267c0dax36 
deploy-2022-11-15T12-02: digest: sha256:97ec7aa503e2d15be85b06da1e8512ca368ccebd46e85505da820b9483ca6fad size: 6988
Deployed Image Tag:  deploy-2022-11-15T12-02
Successfully pushed Docker image to Astronomer registry. Navigate to the Astronomer UI for confirmation that your deploy was successful.

 Access your Deployment: 

 Deployment View: cloud.astronomer-stage.io/ckz7lnmq939963vxtp0qtmu3p/deployments/clai5znlk253991bd26e0ezfdwo/analytics
 Airflow UI: astro01.astronomer-stage.run/d0ezfdwo?orgId=org_FuMJ1ru87w9tyQb6

```

**What CLI Version did you experience this bug?**

**What Operating System is the above CLI installed on?** Linux and MAC

**🪜 Steps To Reproduce**

**📸 Screenshots**

**Comment by sunkickr** [comment]
What is expected?

**Comment by sunkickr** [comment]
deploy is dependent on an env file. Also what command did you run exactly

**Comment by sanchit135** [comment]
Expected is to show some error msg that env file provided is invalid or something like that.

the command I executed is ./astro deploy {deploymentID} -e **invalid** -f

**Comment by kushalmalani** [comment]
@sanchit135 This has been fixed and will be coming out in CLI 1.8 next week
